Bug-Brave: https://github.com/brave/tor_build_scripts/issues/101
Description: Add missing include to unit test to fix Mac builds
Last-Update: 2022-07-06
Origin: https://github.com/openssl/openssl/commit/60f011f584d80447e86cae1d1bd3ae24bc13235b

--- test/v3ext.c
+++ test/v3ext.c
@@ -8,6 +8,7 @@
  */
 
 #include <stdio.h>
+#include <string.h>
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 #include <openssl/pem.h>
